sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModel.cs

```diff
@@ -144,9 +144,15 @@ private static IReadOnlyList<CustomFormSubmodel> ConvertFromUnlabeled(Model mode
 
         private static IReadOnlyList<CustomFormSubmodel> ConvertFromLabeled(Model model)
         {
+            string formType = string.Empty;
+            if (string.IsNullOrEmpty(model.ModelInfo.DisplayName))
+                formType = $"custom-{model.ModelInfo.ModelId}";
```

Comment:
sorry I wrote the wrong thing in my other comment, the service will return this with a colon in RecognizedForm, "custom:{model.ModelInfo.ModelId}"

Reply:
test coming...

Reply:
Added test